coronavirus.md

```diff
@@ -2,18 +2,20 @@
 
 Information for Farset Labs members and visitors during the Coronavirus pandemic, in line with [government guidelines](https://www.executiveoffice-ni.gov.uk/sites/default/files/publications/execoffice/our-approach-to-decision-making-summary-120520.pdf).
 
-#### **Status: Step 3**
+#### **Status: Step 5**
 
-**The space is open to members that went through COVID-19 onboarding and guests under their supervision, but not public events, and all hackerspace meetings are remote.**
+**The space is open to members and small public events with organisers that went through COVID-19 onboarding who can bring in guests under their supervision, and all hackerspace meetings are back in the space with remote participation supported.**
```

Comment:
I think we should define what 'small' means, possibly even being specific to just the event space, for instance 

```suggestion
**The space is open to members and small public events (<20 participants in the Event Space) with organisers that went through COVID-19 onboarding who can bring in guests under their supervision, and all hackerspace meetings are back in the space with remote participation supported.**
```

I'm not "fixed" on the number, just that there *should* be a number

Reply:
Ok, we should go for 15 people in that case - that is the maximum number before an additional risk assessment is required.